### ISSUE TITLE
Could Citus support multi-coordinator architecture

### ISSUE THREAD

**Issue by ChenHuajun** [citusdata/citus#3762]
Hi, all

There is currently only one Coordinator node in the Citus cluster. When the Coordinator is overloaded, Citus MX is a solution. However, using Citus MX has the following problems.

1. The MX worker distributes SQL and stores data. The computing power and storage capacity cannot be scaled out independently.
2. For Citus cluster with many nodes, too many MX workers increase the burden of metadata synchronization
3. If only some workers are used as MX workers, since these MX workers also store data, the load will be higher than other workers, and the load is unbalanced.

Could consider developing a multi-coordinator architecture, or just let MX workers support usage modes that do not store data (data from distribution tables and reference tables)?

**Comment by onderkalaci** [comment]
> I believe this is largely handled in Citus 11, see https://www.citusdata.com/blog/2022/03/26/test-drive-citus-11-beta-for-postgres/#section3.
> 
> Only schema change operations now need to go through the co-ordinator, which is unlikely to overload a single node (most people at least make sparse schem changes)

yes, indeed this is largely resolved in Citus 11.0, so closing the issue.

**Comment by JelteF** [comment]
Also if you really want nodes that can only do queries, but don't have any shards themselves. You can use 
```sql
SELECT citus_set_node_property(hostname, port, 'shouldhaveshards', false).
```
